File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Avslutning.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class Avslutning {

    private String dato;

    private Aarsak aarsak;

    private String type;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Avslutning.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class Avslutning(
    var dato: String? = null,
    var aarsak: Aarsak? = null,
    var type: String? = null
)
